### DISCUSSION TITLE
Change default Example admonition color for slate design

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2359] [Ideas]
I personally think that the colour used for the example type of admonition should be tweaked a bit, especially for the slate scheme.

When you look at it does the icon not stick out that well compared to the other types.
![image](https://user-images.githubusercontent.com/11576465/109405001-1030e200-796c-11eb-9acd-b139ee3c5cee.png)

This works great on the white theme, but on slate, which I assume a fair amount of people is probably using, does it not show that great.
I can't give a suggestion about what colour should be used instead... Perhaps a brighter purple would already be enough.

**Comment by squidfunk** [comment]
Thanks for reporting. I switched from the `a400` color shade to `a200` in 71926399 – that's the lightest we could go without breaking out of the Material Design accent color palette system. It now looks like this:

<img width="716" alt="Bildschirmfoto 2021-02-28 um 08 38 57" src="https://user-images.githubusercontent.com/932156/109411172-6e77b800-79a0-11eb-9dd7-0c72753acf65.png">

**Reply by Andre601** [reply]
Looks a lot better.
